sql/tables.sql

```diff
@@ -0,0 +1,191 @@
+CREATE TABLE public.githuborgs (
+    org_id integer NOT NULL,
+    org_name text NOT NULL
+);
+
+
+ALTER TABLE public.githuborgs OWNER TO repolinter;
+
+--
+-- Name: TABLE githuborgs; Type: COMMENT; Schema: public; Owner: repolinter
+--
+
+COMMENT ON TABLE public.githuborgs IS 'contains org names on github.';
+
+
+--
+-- Name: COLUMN githuborgs.org_id; Type: COMMENT; Schema: public; Owner: repolinter
+--
+
+COMMENT ON COLUMN public.githuborgs.org_id IS 'org_ids';
+
+
+--
+-- Name: COLUMN githuborgs.org_name; Type: COMMENT; Schema: public; Owner: repolinter
+--
+
+COMMENT ON COLUMN public.githuborgs.org_name IS 'org names';
+
+
+--
+-- Name: github-orgs_org_id_seq; Type: SEQUENCE; Schema: public; Owner: repolinter
+--
+
+CREATE SEQUENCE public."github-orgs_org_id_seq"
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1;
+
+
+ALTER TABLE public."github-orgs_org_id_seq" OWNER TO repolinter;
+
+--
+-- Name: github-orgs_org_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: repolinter
+--
+
+ALTER SEQUENCE public."github-orgs_org_id_seq" OWNED BY public.githuborgs.org_id;
+
+
+--
+-- Name: githubrepos; Type: TABLE; Schema: public; Owner: repolinter
+--
+
+CREATE TABLE public.githubrepos (
+    repo_name text NOT NULL,
+    full_name text NOT NULL,
+    github_owner_details json,
+    collaborators_url text NOT NULL,
+    teams_url text,
+    updated_at date,
+    git_url text,
+    ssh_url text,
+    clone_url text NOT NULL,
+    language text,
+    watchers_count integer,
+    open_issues_count integer NOT NULL,
+    license json,
+    org_name text,
+    github_id integer,
+    repolinter_scan text
+);
+
+
+ALTER TABLE public.githubrepos OWNER TO repolinter;
+
+--
+-- Name: TABLE githubrepos; Type: COMMENT; Schema: public; Owner: repolinter
+--
+
+COMMENT ON TABLE public.githubrepos IS 'all repos from the amazon orgs';
+
+
+--
+-- Name: COLUMN githubrepos.repo_name; Type: COMMENT; Schema: public; Owner: repolinter
+--
+
+COMMENT ON COLUMN public.githubrepos.repo_name IS 'repository name';
```

Comment:
You can delete the comment lines here and elsewhere; it's just noise from the SQL dump.